NBitcoin/Payment/BitcoinUrlBuilder.cs

```diff
@@ -47,38 +47,25 @@ public BitcoinUrlBuilder(string uri, Network network)
 				throw new ArgumentNullException(nameof(uri));
 			if (network == null)
 				throw new ArgumentNullException(nameof(network));
-			scheme = network.UriScheme;
-			if (!uri.StartsWith($"{scheme}:", StringComparison.OrdinalIgnoreCase))
-			{
-				if (uri.StartsWith("bitcoin:", StringComparison.OrdinalIgnoreCase))
-					scheme = "bitcoin";
-				else
-					throw new FormatException("Invalid scheme");
-			}
+
+			var parsedUri = new Uri(uri, UriKind.Absolute);

```

Comment:
`UriFormatException` is derived from `FormatException`, so it must not error handling for any customer. The message is different, but it can be addressed through the `TryCreate` method.

Still there can be other issues with `uri` which `Uri` type can handle better than the custom logic.

Reply:
Added it to the description.